internal/log/log.go

```diff
@@ -32,7 +32,8 @@ import (
 
 var (
 	// Log is the base logger
-	Log logr.Logger = logr.New(ctrllog.NullLogSink{})
+	Log  logr.Logger = logr.New(ctrllog.NullLogSink{})
+	Sync             = zapcore.WriteSyncer(nil)
```

Comment:
I could also refactor and add a `LogWrapper` type, holding them together but I thought this was the least intrusive change

Reply:
If you make this one `io.Writer` too here, then you could wire a `bytes.Buffer` here. Or just have it `nil` directly.